pinot-common/src/main/java/org/apache/pinot/common/function/FunctionRegistry.java

```diff
@@ -46,11 +50,7 @@ private FunctionRegistry() {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(FunctionRegistry.class);
 
-  // TODO: consolidate the following 2
-  // This FUNCTION_INFO_MAP is used by Pinot server to look up function by # of arguments
-  private static final Map<String, Map<Integer, FunctionInfo>> FUNCTION_INFO_MAP = new HashMap<>();
```

Comment:
in order to delete this we need to stop supporting random `_` added inside function name b/c `canonicalize(functionName)` will remove it. 

we can keep this but mark as deprecated